connector_magento_configurable/models/configurable.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import logging
+
+from odoo import models, fields, tools
+
+from odoo.addons.component.core import Component
+
+_logger = logging.getLogger(__name__)
+
+
+class ConfigurableBatchImporter(Component):
+    """ Import the Configurable Products.
+
+    For every Configurable Product not yet converted from flat
+    to templated product, creates a job
+    """
+    _name = 'magento.product.configurable.batch.importer'
+    _inherit = 'magento.delayed.batch.importer'
+    _apply_on = ['magento.product.configurable']
+
+    def run(self, filters=None):
+        """ Run the synchronization """
+        from_date = filters.pop('from_date', None)
+        # to_date = filters.pop('to_date', None)
+        criterias = [('product_type', '=', 'configurable')]
+        if from_date:
+            criterias.append(('write_date', '>', from_date.strftime(
+                tools.misc.DEFAULT_SERVER_DATETIME_FORMAT)))
+        internal_ids = self.env['magento.product.product'].search(criterias)
```

Comment:
preferable to give another name than something_ids when it holds records and not ids :)